
bool flag2d
float32 xi
float32 yi
float32 radius_x
float32 radius_y
float32 radius_mean
float32 angle

bool flag3d
geometry_msgs/Pose pose

float32 dice_number
